code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -173,7 +173,7 @@
 
 /datum/objective/traitor_objectives
 	name = "traitor objective"
-	explanation_text = "Complete objectives colletively worth more than %REPUTATION% reputation points"
+	explanation_text = "Complete objectives colletively worth more than %REPUTATION% reputation points."
```

Comment:
If we're refactoring this to add a period, they spelled `collectively` wrong too :)

Reply:
Oh, good catch, I didn't notice that.